### DISCUSSION TITLE
Concerns About Accessibility Compliance in PrimeNG UI Kit

### DISCUSSION THREAD

**Discussion by Dinaaaaaaa-tr** [primefaces/community#3033] [Figma UI Kit]
I am a UI/UX designer, and my team is considering purchasing the PrimeNG UI Kit. We were particularly interested because of the claim that it follows accessibility guidelines (e.g., WCAG standards). However, after conducting a small test, we identified several areas where the UI Kit doesn’t seem to fully comply with basic accessibility principles, which is a significant concern for us. For example here are some of the specific issues we encountered:

- Contrast issues
- Missing Alt Text and Description Text in icons and images accordingly
- Placeholders in inputs

I would like to know if your team conducts accessibility tests on the PrimeNG components and UI Kit to ensure they comply with WCAG standards. If not, are there any plans to implement these tests to ensure the library is fully accessible in future updates?

Thank you in advance

**Comment by onursenture** [comment]
Hello, We prioritize WCAG standards to ensure accessibility in our libraries, but sometimes, we need to be flexible with these standards when it comes to theming. We offer several presets, including the Nora preset, which we believe effectively meets contrast requirements. As you explore the new PrimeNG, we recommend evaluating the Nora preset.

Currently, our UI Kit only includes the Aura preset, which may have some accessibility gaps.

We value your feedback. If you notice any clear accessibility violations rather than design decisions, please let us know so we can address them in future versions.

Thank you!
